distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestRegistry.java

```diff
@@ -87,6 +87,8 @@
 
     private static final String REMOTE_METACARD_ID = "09876543210987654321098765432100";
 
+    private static final long SLEEP_TIME = 2000;
```

Comment:
Most places have the sleep time at 1 second but i went with 2 here to reduce thrashing. Figured if the system is really slow and our stuff isn't coming up in time it might be better to reduce the frequency of these checking threads.